bitcoind-regtest/src/Bitcoin/Core/Regtest/Generator.hs

```diff
@@ -408,8 +408,8 @@ easySpendScript = Script . maybe mempty mkScript
 easySpendAddress :: Maybe Word8 -> Text
 easySpendAddress ix = addr
   where
-    Just addr =
-        H.addrToText H.btcRegTest =<< (H.outputAddress . H.toP2SH) (easySpendScript ix)
+    addr =
+        fromJust $ H.addrToText H.btcRegTest =<< (H.outputAddress . H.toP2SH) (easySpendScript ix)
```

Comment:
This is a lateral move.  I prefer suppressing the warning to making this change.  Same below.

Reply:
fixed.